engine/tools/wscript

```diff
@@ -45,8 +45,10 @@ def configure(conf):
         conf.env.append_value('LINKFLAGS', ['-framework', 'Cocoa', '-framework', 'OpenGL', '-framework', 'AGL', '-framework', 'IOKit', '-framework', 'Carbon', '-framework', 'CoreVideo', '-framework', 'QuartzCore'])
     elif platform == 'x86_64-ios':
         pass
-    elif platform in ["x86_64-linux", "arm64-linux"]:
+    elif platform == 'x86_64-linux':
         conf.env['LIB_X'] = ['Xext', 'X11', 'Xi', 'GL', 'GLU']
+    elif platform == 'arm64-linux':
+        conf.env['LIB_X'] = ['Xext', 'X11', 'Xi', 'EGL', 'GLESv1_CM', 'GLESv2']
```

Comment:
(old code, I know)
The "X" library shouldn't need these extra GL libraries.
Instead, they should go into "LIB_OPENGLES" (although that is also already setup in waf_dynamo.py)
Our wscripts have tons of small issues like that.

Reply:
I apologise - I overlooked this change. As with other wscripts, `LIB_X` is no longer needed for Linux, so I removed it for both platforms. And as I understand it, if the engine builds with all the tests after these deletions, then it's ok.